JS/Planes/experimentalPlane.js

```diff
@@ -1,68 +1,64 @@
-const Plane = require('./Plane');
```

Comment:
Please rename file to have it started with capital letter.

Reply:
Done